src/domain/user-group/user-group.resolver.fields.ts

```diff
@@ -22,4 +23,18 @@ export class UserGroupResolverFields {
   async parent(@Parent() userGroup: UserGroup) {
     return await this.userGroupService.getParent(userGroup);
   }
+
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.